conbench/templates/compare-list.html

```diff
@@ -253,45 +223,5 @@ <h4>Compare current contender run with another run:</h4>
           },
         });
 
-    // Note(JP): maybe this is for the "compare current contender run to
-    // another run" form. Who uses that? Why is this workflow important?
```

Comment:
Ah :)

Thanks for pulling through. Big fan of removing UX complexity.

We can add things like that back if we ever want to.